tach/cli.py

```diff
@@ -207,15 +207,23 @@ def parse_arguments(
 ) -> tuple[argparse.Namespace, argparse.ArgumentParser]:
     parser = build_parser()
     parsed_args = parser.parse_args(args)
+
+    if len(args) < 1 or args[0] not in ["pkg", "clean", "sync"]:
+        # TODO: unify project config handling
+        fs.validate_project_config_path()
+
     return parsed_args, parser
 
 
 def tach_check(
+    exact: bool = False,
     root: str = ".",
-    strict: bool = False,
     exclude_paths: Optional[list[str]] = None,
 ):
     try:
+        project_config = parse_project_config()
+        if exact is False and project_config.exact is True:
+            exact = True
         project_config = parse_project_config(root=root)
```

Comment:
This got tweaked in main:
```suggestion
        project_config = parse_project_config(root=root)
        if exact is False and project_config.exact is True:
            exact = True
```